data/json/items/containers.json

```diff
@@ -475,16 +457,16 @@
         "type": "CONTAINER",
         "symbol" : "}",
         "color" : "light_cyan",
-        "name" : "little metal tank",
+        "name" : "metal tank (2L)",
         "category": "other",
-        "description" : "A little metal tank for holding gas or liquids.  Useful for crafting.",
+        "description" : "A small metal tank for gas or liquids.  Useful for crafting.",
         "price" : 1000,
         "weight" : 800,
-        "volume" : 5,
+        "volume" : 8,
```

Comment:
There are some items which uncraft to little metal tanks. Those should probably be adjusted too, possibly by having them uncraft to something else.